oxo.py

```diff
@@ -1,26 +1,195 @@
 class OxoBoard:
-    def __init__(self):
-        """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+
+    PLAYER_y = 0
+    PLAYER_X = 1
+
+    def __init__(self, grid_width=3, grid_height=3, win_row_amount=3):
+        """ The initialiser. Initialise any fields you need here.
+
+        :param grid_width:     width of board
+        :param grid_height:    height of board
+        :param win_row_amount:
+        """
+        self.grid_size = {"width": grid_width, "height": grid_height}
```

Comment:
A dictionary probably isn't the best choice of data structure here -- I'd recommend using a 2-tuple `(grid_width, grid_height)` or two fields `self.grid_width` and `self.grid_height`